redis/src/config.rs

```diff
@@ -124,7 +124,8 @@ impl Default for Config {
     }
 }
 
-/// This is a 1:1 copy of the [`redis::ConnectionAddr`] enumeration.
+/// This is a 1:1 copy of the [`redis::ConnectionAddr`] enumeration (excluding `tls_params` since it is entirely opaque to consumers).
```

Comment:
Are you sure about that? Maybe the `tls_params` should be exposed by our config struct as well? I feel a bit uneasy leaving out configuration fields. Even if those fields can't be deserialized with serde it should be possible to set the `tls_params` after deserialization in code.

Reply:
Why I believe that it's an oversight is that those fields here are exposed: <https://github.com/redis-rs/redis-rs/blob/3e6dc2ef611060a665d8e5d4c14bd8618517fac2/redis/src/lib.rs#L468-L469>

But yet the actual `TlsConnParams` is kept opaque.

Reply:
I created an issue on their issue tracker to hopefully clarify this: https://github.com/redis-rs/redis-rs/issues/1006